core/src/main/scala/org/broadinstitute/dsde/rawls/entities/local/EntityStatisticsCacheSupport.scala

```diff
@@ -61,6 +61,7 @@ trait EntityStatisticsCacheSupport extends LazyLogging with RawlsInstrumented {
       case (typeName, typeMetadata) => typeName -> typeMetadata.attributeNames.map(AttributeName.fromDelimitedName)
     }
     val timestamp: Timestamp = new Timestamp(workspaceContext.lastModified.getMillis)
+    opportunisticEntityCacheSaveCounter.inc()
```

Comment:
q: Do we want to record the metric in the Success case on line 69, so we're reporting when we try & succeed at saving? Or do we want to record the metric every time we try to opportunistically save (even when it fails)

Reply:
Upon initial thought I think succeeded cache saves would hold more value, but that opinion is subject to change if other people dissent. Regardless though, there needs to be consistency with the `entityCacheSaveCounter` counter. The plan was for the two counters to not overlap/get tripped by the same cache save request, but also for the union of the two to cover every cache update.

So more broadly either exactly one of counters moves to another location in the code to keep the aforementioned consistency or we instead implement a counter that covers everything and another one that resides in a subset of the other counter, which one sounds better do you think?

Reply:
+1 for keeping the current counters and setting it up so exactly one counter gets tripped-

Reply:
Got it, I'll move the incrementing to line 69. Relating to that though, is the counter for non-opportunistic saves in `EntityStatisticsCacheMonitor.scala` sufficient enough to uphold the consistency discussed in your first comment?